config/cmake/HDFFortranCompilerFlags.cmake

```diff
@@ -57,6 +57,15 @@ if (CMAKE_Fortran_COMPILER_ID STREQUAL "NAG")
     list (APPEND HDF5_CMAKE_Fortran_FLAGS "-ieee=full")
 endif ()
 
+if (CMAKE_Fortran_COMPILER_ID STREQUAL "NVHPC")
+  set (CMAKE_Fortran_FLAGS "${CMAKE_Fortran_FLAGS} -fPIC")
```

Comment:
This might not be needed as CMake should handle this flag.

Reply:
Yes, it does